src/plugins/discord/fetcher.js

```diff
@@ -148,7 +124,7 @@ export class Fetcher implements DiscordFetcher {
     const endpoint = `/channels/${channel}/messages/${message}/reactions/${emojiRef}?after=${after}&limit=${reactionsLimit}`;
     const response = await this._fetch(endpoint);
     const results = response.map((x) => ({
-      emoji: x.emoji,
+      emoji: emoji,
```

Comment:
@BrianLitwin can you explain this change?

Reply:
Yeah. We are asking for the users that responded with a particular emoji here, and the emoji isn't present in the response, but is passed in as a parameter to the Discord fetch. 

I need to follow up and update the unit tests in few places to reflect this. Thanks for the catch.